forte/data/data_store.py

```diff
@@ -247,6 +247,18 @@ def _new_group(self, type_name: str, member_type: str) -> List:
         entry += len(self._type_attributes[type_name]) * [None]
         return entry
 
+    def _is_annotation(self, type_name: str) -> bool:
+        r"""This function takes a type_id and returns whether a type
+        is an annotation type or not.
+        Args:
+            type_name (str): The name of type in `self.__elements`.
+        Returns:
+            A boolean value whether this type_id belongs to an annotation
+            type or not.
+        """
+        entry_class = get_class(type_name)
+        return issubclass(entry_class, Annotation)
+
     def add_annotation_raw(self, type_name: str, begin: int, end: int) -> int:
         r"""This function adds an annotation entry with ``begin`` and ``end``
         indices to the sortedlist of ``type_name`` in ``self.__elements``,
```

Comment:
Referring the internal implementation is not good for docstring. But if this is for internal only, than you should make this function private.